workflow-templates/im-test-k6-ci.yml

```diff
@@ -0,0 +1,24 @@
+name: K6 Automated Load Tester
+
+on: 
+  push:
+    branches: 
+      - master
+      - main
+
+env:
+  testFile: "tests/k6/check-health-simple.js"
+      
+jobs:
+  k6_test:
+    runs-on: [self-hosted, ubuntu-20.04]
+    container: docker://loadimpact/k6:latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    
+    - name: Run k6 test - docker
+      working-directory: tests/k6
+      run: |
+        k6 run ${{env.testFile}} --vus 1 -i 1 --insecure-skip-tls-verify -out JSON="./k6-output.json"
```

Comment:
I would combine this with the other one and just use commented out items with TODOs to indicate the different options.

Reply:
Combined files.